src/commands/script/execute/simulate.ts

```diff
@@ -56,7 +56,8 @@ export const simulateExecute = async (
       transport: http(FORK_CHAIN_URL, { timeout: 60_000 }),
     })
 
-    const publicClient = createPublicClient({ chain: chainInfo.viemChain.instance, transport: http(FORK_CHAIN_URL) })
+    const forkClient = createPublicClient({ chain: chainInfo.viemChain.instance, transport: http(FORK_CHAIN_URL) })
+    const publicClient = createPublicClient({ chain: chainInfo.viemChain.instance, transport: http(getProjectRpc()) })
```

Comment:
Could you help add the comment to elaborate why adding the `forkClient` solves the wrong gas estimation for the future references please? It's kinda not obvious here.